<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <widget class="QWidget" name="centralWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>581</width>
     <height>321</height>
    </rect>
   </property>
   <widget class="QPushButton" name="relog_button">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Log in</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="high_ping_username">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>100</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="mm_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="high_ping_button">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>100</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="whitelist_country">
    <property name="geometry">
     <rect>
      <x>372</x>
      <y>160</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="whitelist_country_button">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>160</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="config_info">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>210</y>
      <width>191</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="matchmaking_info">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>331</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="in_match">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>In match:</string>
    </property>
   </widget>
   <widget class="QLabel" name="in_match_value">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>False</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quit_mm_button">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logout_button">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>30</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="logged_in_label">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Logged in</string>
    </property>
   </widget>
   <widget class="QLabel" name="high_ping_label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>80</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>High ping:</string>
    </property>
   </widget>
   <widget class="QLabel" name="whitelist_country_label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>140</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Whitelist country:</string>
    </property>
   </widget>
   <widget class="QLabel" name="high_ping_tooltip">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>80</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
   <widget class="QLabel" name="whitelist_country_tooltip">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>140</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
